src/portfft/dispatcher/subgroup_dispatcher.hpp

```diff
@@ -719,7 +793,7 @@ struct committed_descriptor_impl<Scalar, Domain>::num_scalars_in_local_mem_struc
     Idx num_scalars_per_sg = detail::pad_local(2 * dft_length * n_ffts_per_sg, 1);
     Idx max_n_sgs = (desc.local_memory_size - twiddle_bytes) / static_cast<Idx>(sizeof(Scalar)) / num_scalars_per_sg;
     num_sgs_per_wg = std::min(Idx(PORTFFT_SGS_IN_WG), std::max(Idx(1), max_n_sgs));
-    Idx res = num_scalars_per_sg * num_sgs_per_wg;
+    Idx res = detail::pad_local(2 * dft_length * n_ffts_per_sg * num_sgs_per_wg, 1);
```

Comment:
Was this the off-by-1 bug you were referring to?

Reply:
yes! I've added a wee comment